test/unit/views/RegisteredAddressForEoriViewSpec.scala

```diff
@@ -46,6 +49,18 @@ class RegisteredAddressForEoriViewSpec extends QuestionViewBehaviours[Registered
       routes.RegisteredAddressForEoriController.onSubmit(NormalMode).url,
       "field1", "field2", "field3", "field4", "field5"
     )
+
+    "show the expected text" in {
+      val text = asDocument(createViewUsingForm(form)).text()
+
+      text must include("What is the registered name and address for EORI number eori-789012?")
```

Comment:
this is failing currently
It shows `What is the registered name and address for EORI number {0}`

Any idea on how to fix it?